package.json

```diff
@@ -47,7 +47,7 @@
     "puppeteer": "^10.2.0",
     "serve": "^11.3.2",
     "simple-git": "^2.24.0",
-    "testcafe": "^1.15.0",
+    "testcafe": "^1.17.0",
```

Comment:
should this be 1.17.1?

Reply:
Based on the package-lock, it looks like 1.17.1 is the version which is actually being used. I can bump it here as well just to be safe